### PR TITLE
nvme: Add more BDNVMEControllerInfo fields

### PR DESCRIPTION
**Pull request by tbzatek**
Broader selection of fields as needed for UDisks.

### CODE REVIEW

**Review by vojtechtrefny — APPROVED** [review]
Looks good to me.

File: tests/nvme_test.py [hunk 1]
```diff
@@ -105,7 +105,18 @@ def test_ctrl_info(self):
...
+        self.assertFalse(info.features & BlockDev.NVMEControllerFeature.MGMT_PCIE)
+        self.assertFalse(info.features & BlockDev.NVMEControllerFeature.MGMT_SMBUS)
         self.assertEqual(info.fguid, "")
+        self.assertEqual(info.pci_vendor_id, 0)
+        self.assertEqual(info.pci_subsys_vendor_id, 0)
+        self.assertIn("Linux", info.model_number)
+        self.assertGreater(len(info.serial_number), 0)
+        self.assertGreater(len(info.firmware_ver), 0)
+        self.assertGreater(len(info.nvme_ver), 0)
+        self.assertEqual(info.controller_type, BlockDev.NVMEControllerType.UNKNOWN)
```
Comment (vojtechtrefny) [inline tests/nvme_test.py:119]: Failed tests suggest `BlockDev.NVMEControllerType.IO` should be here instead of `UNKNOWN`.

Comment (tbzatek) [reply tests/nvme_test.py:119]: Ahh, another kernel nvme target difference. Let's just remove this test.

**Comment by tbzatek** [comment]
Jenkins, test this please.

**Pull request merged**